### PR TITLE
filet project split tvl to multiple chains

### PR DESCRIPTION
**Pull request by woshidama323**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/filet: 
        
 
  
    
    
    Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x64F23Eb153D7496Fc31C1b38a41068995354773E","data":"0x63f9d8660000000000000000000000000000000000000000000000000000000000000063","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
        at Array.forEach (<anonymous>)

**Comment by max1015070108** [comment]
Hi But it works well on my dev local with
`node test projects/filet/index.js`

`tvlData: 17868.769271
--- bsc ---
FIL                       547.22 k
Total: 547.22 k 

--- mixin ---
FIL                       77.37 k
Total: 77.37 k 

--- heco ---
FIL                       7.63 k
Total: 7.63 k 

--- filecoin ---
FIL                       514.25 k
Total: 514.25 k 

--- tvl ---
FIL                       1.15 M
Total: 1.15 M 

------ TVL ------
bsc                       547.22 k
mixin                     77.37 k
heco                      7.63 k
filecoin                  514.25 k

total                    1.15 M`

**Comment by llamatester** [comment]
The adapter at projects/filet exports TVL: 
        
 
 
    bsc                       544.69 k
    mixin                     77.02 k
    heco                      7.82 k
    filecoin                  3.19 M
    
    total                    3.82 M

**Comment by llamatester** [comment]
The adapter at projects/filet exports TVL: 
        
 
 
    bsc                       544.69 k
    heco                      7.82 k
    mixin                     77.02 k
    filecoin                  3.19 M
    
    total                    3.82 M

**Comment by g1nt0ki** [comment]
@woshidama323  sorry, been bit busy, will review this tomorrow

File: projects/filet/index.js [hunk 1]
```diff
@@ -1,10 +1,91 @@
...
+const filetNativeCon = "0xBfd5BB742940a5b059710F4d1216db1F1287F4e0";
+
+const filetStakingCon_HECO = "0x62C7aaCdCCEc32A2E33d5fa535B374f8a4a42566";
+const filetStakingConFlexible_HECO = "0x64F23Eb153D7496Fc31C1b38a41068995354773E"
+
+const poolOnFVM180 = 4010
+const poolOnFVM360 = 4011
+
+// filet 
+const filetAPI = "https://api.filet.finance/pledge/ext/tx/pledgeTxAll"
```
Comment (g1nt0ki) [inline projects/filet/index.js:27]: unable to test, this api is returning 502 :(

Comment (woshidama323) [reply projects/filet/index.js:27]: Sorry for inconvenient
Our dev teams have an upgrading plan for server, It works normally now,
Please try again

**Comment by woshidama323** [comment]
@g1nt0ki  Could you help to try again since it works normally now

**Pull request merged**